src/main/java/org/devnull/jedi/configs/JediConfig.java

```diff
@@ -72,6 +72,11 @@
 	public String unix_socket_path = null;
 
 	/**
+	 * Timeout value for the unix socket. Without this sockets get stuck in the CLOSE_WAIT status and file descriptors leak.
```

Comment:
When adding javadoc for time-related parameters, always include the unit of measure.  This should explicitly say "timeout in milliseconds".